python/astro_metadata_translator/translators/megaprime.py

```diff
@@ -91,9 +95,15 @@ def to_location(self):
         location : `astropy.coordinates.EarthLocation`
             An object representing the location of the telescope.
         """
-        # Height is not in MegaPrime files. Use the value from EarthLocation.of_site("CFHT")
-        value = EarthLocation.from_geodetic(self._header["LONGITUD"], self._header["LATITUDE"], 4215.0)
-        self._used_these_cards("LONGITUD", "LATITUDE")
+        # Height is not in some MegaPrime files. Use the value from EarthLocation.of_site("CFHT")
+        # Some data uses OBS-LONG, OBS-LAT, other data uses LONGITUD and LATITUDE
+        for long_key, lat_key in (("LONGITUD", "LATITUDE"), ("OBS-LONG", "OBS-LAT")):
+            if long_key in self._header and lat_key in self._header:
+                value = EarthLocation.from_geodetic(self._header[long_key], self._header[lat_key], 4215.0)
+                self._used_these_cards(long_key, lat_key)
+                break
+        else:
+            value = EarthLocation.of_site("CFHT")
         return value
 
     def to_detector_num(self):
```

Comment:
I was trying not to write repetitive docstrings when nothing is different to the base class.  If you use pydoc or sphinx the documentation will turn up.  @jonathansick I think I recall a conversation somewhere where we said it was okay to inherit docstrings.

Reply:
I wondered about this myself the other day, in a different context.

I reckon that for much of the codebase, at least while we're in development, the primary consumer of the docstring will be somebody who is directly modifying or — at least — looking at the code. Directing them to some other tool in order to read the API documentation seems unfortunate, so my instinct is that repetitive docstrings are actually helpful.

At the same time, though, I'm as leery of copying & pasting as the next person, so I'm not totally happy with that answer.

I'd definitely be interested in hearing ideas on best practice from other projects, although I do caution that we should use them as guidance rather than adopt them wholesale.

Reply:
For the translators embedded in the package I think it's reasonable for someone working on the package to understand what the standard properties in `ObservationInfo` mean without having to copy and paste everything (the docstrings are automatically constructed in the base class from the property definitions).  The documentation for a particular translation doesn't help much then.  If something a bit interesting is happening then I agree that a docstring should be added.  If the consensus is that I should always add them then I will do so.  It's probably the case that if a translator is external to this package it would be clearer to always add the docstring.

Reply:
I would generally prefer not to duplicate docstrings.  A code comment indicating that a valid docstring can be found in a particular base class might be a good idea.  If you're reading the docs by looking at the code itself, you're probably in a position to easily find docs in another file already; if you wanted effortless navigation you'd be reading the HTML.

Reply:
> A code comment indicating that a valid docstring can be found in a particular base class might be a good idea
I very much like this idea.

Reply:
Putting in a code comment was our conclusion in Slack. I'll make a point of getting this into the Developer Guide.